src/rgw/setup/templates/logrotate.service.tmpl

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+/usr/sbin/logrotate /etc/logrotate.d/radosgw
+EXITVALUE=$?
+if [ $EXITVALUE != 0 ]; then
+    /usr/bin/logger -t logrotate "ALERT exited abnormally with [$EXITVALUE]"
+fi
+exit $EXITVALUE
```

Comment:
#L3, 

> /usr/sbin/logrotate /etc/logrotate.d/radosgw

This line will enable logrotate on hourly basis for only radosgw service.
It will not impact other components log rotation.